src/pdf_beamtime/src/pdf_beamtime_server.cpp

```diff
@@ -47,9 +47,10 @@ PdfBeamtimeServer::PdfBeamtimeServer(
     std::bind(&PdfBeamtimeServer::handle_cancel, this, _1),
     std::bind(&PdfBeamtimeServer::handle_accepted, this, _1));
 
-  // // Initialize to home
+  // Initialize to home
   current_state_ = State::HOME;
   gripper_present_ = node_->get_parameter("gripper_present").as_bool();
+  inner_state_machine_ = new InnerStateMachine(node_, State::HOME);
```

Comment:
I'm still a little unsure why the InnerStateMachine needs to know about the external state? Is this just for logging?

Reply:
In the earlier version before separating the internal state machine from the external one, I used the external state to control internal state logic. After the separation., right now it is only used for logging.

Reply:
InnerStateMachine only accepts node_ as its sole argument. the external state is removed. Respective changes were made to inner_state_machine.hpp and README.md
https://github.com/maffettone/erobs/blob/e33193c3e57f8ec62e71e275e81b4669b41f0edb/src/pdf_beamtime/src/pdf_beamtime_server.cpp#L53
https://github.com/maffettone/erobs/blob/e33193c3e57f8ec62e71e275e81b4669b41f0edb/src/pdf_beamtime/src/inner_state_machine.cpp#L5-L7
https://github.com/maffettone/erobs/blob/e33193c3e57f8ec62e71e275e81b4669b41f0edb/src/pdf_beamtime/README.md